src/screens/SettingsScreen.tsx

```diff
@@ -30,6 +30,27 @@ import FontSelector from "../widgets/FontSelector";
 import Select from "../widgets/Select";
 import { RootStackParamList } from "../RootStackParamList";
 
+type AnchorButtonProps = {
+  onPress: () => void;
+  children: React.ReactNode;
+};
+
+function AnchorButton(props: AnchorButtonProps) {
+  const { children, onPress } = props; 
+  const theme = useTheme();
+
+  return (
+    <Button
+      mode="contained"
+      color={theme.colors.accent}
+      labelStyle={{ color: (theme.dark) ? "#000000" : "#ffffff" }}
```

Comment:
Here however I can complain (like what I said above). This should be in the theme and used here and with the FAB.